packages/sdk/src/types/runtime/actor.ts

```diff
@@ -345,47 +304,73 @@ export class Actor implements ActorLike, Patchable<ActorLike> {
 	 * Adds a collider of the given type and parameters on the actor.
 	 * @param colliderType Type of the collider to enable.
 	 * @param isTrigger Whether the collider is a trigger volume or not.
+	 * @param radius The radius of the collider.
 	 * @param center The center of the collider, or default of the object if none is provided.
-	 * @param radius The radius of the collider, or default bounding if non is provided.
 	 */
 	// * @param collisionLayer The layer that the collider operates in.
 	public setCollider(
 		colliderType: 'sphere',
 		// collisionLayer: CollisionLayer,
 		isTrigger: boolean,
-		center?: Vector3Like,
-		radius?: number): void;
+		radius: number,

```

Comment:
>radius: number, [](start = 2, length = 15)

We discussed this that the Unity client provides a reasonable default for this when specified, and that optional should be allowed for the size related properties here.  These should still be optional.

Reply:
Even if that default is a constant? The more we let Unity specify defaults, the less useful intellisense becomes. Letting Unity set box collider defaults makes sense because it fits it to the mesh, but I don't think Unity computes a bounding sphere for meshes.

---
In reply to: [315736855](https://github.com/microsoft/mixed-reality-extension-sdk/pull/415#discussion_r315736855) [](ancestors = 315736855)